TreeSearch.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Scanner;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class TreeSearch {
+
+	Node root;
+
+	public void TreeQueue(Node root) { //queue traversal
+		Queue<Node> q = new LinkedList<Node>();
+
+		if (root == null) {
+			return;
+		}
```

Comment:
When are the times you are going to put in a null root? You don't need to write this to run this code, but if you want, just to be extra cautious, then do it.